app/views/dashboard/index.html.haml

```diff
@@ -34,6 +34,14 @@
     = render partial: "dashboard/ng-partials/widgets/schedule_sequence"
   %script{:id => "widgets/calendar", :type => "text/ng-template"}
     = render partial: "dashboard/ng-partials/widgets/calendar"
+  %script{:id => "widgets/move", :type => "text/ng-template"}
```

Comment:
Ok, that makes sense. I'll switch it over to rails partials.

I was doing it not only for organizational purposes, but because I imagine the user one day being able to 'configure' their dashboard by adding/removing/moving widgets however they like. For that to work it seems like the widgets would have to be separated in some way and then mashed together on the fly like this. Plus, it makes it super easy to move these things around from a code standpoint.